### ISSUE TITLE
Counters data fetch

### ISSUE THREAD

**Issue by H0fek** [KLFAMO/web_famo_test#1]
New endpoint /api/counters

GET action:
- params in url: startMjd, endMjd, resourceName 
   - ex. `/api/counters?startMjd=1000&endMjd=1001&resourceName=counter1`
- response is JSON object with MJD and Value array fields
  - ex. 
```
{
  1 : 10.1,
  2 : 20.2,
  3 : 30.3
}
```

- if no params are supplied then return last value for each resource
  - ex. 
```
{
  "counter1": 1,
  "counter2": 2,
  "counter3": 3
}
```

**Comment by KLFAMO** [comment]
Let's talk about it, some things are not clear to me. Probably we have to specify counters names (we may have different counters in labs) and channel (each counter hase usually several channels).
I am a bit confused about ex. What is 1, 2, 3? Is it MJD?  I don't see array in json example.
Maybe you mean something like:
``[
  { "MJD": 1, "VAL": 10.1 },
  { "MJD": 2, "VAL": 20.2 },
  { "MJD": 3, "VAL": 30.3 }
]
``

**Comment by H0fek** [comment]
Yes, I meant the MJD : VALUE and you're right, my mistake. Of course it's even not a proper JSON format. 

What about this one:
```
{
  "MJD": [
    6000,
    6001,
    60002
  ],
  "Values": [
    123456789,
    345678901,
    789123456
  ]
}
```
It's similar to how you handle data in JS currently. This way we have an object which is open for further modifications in future.

**Comment by KLFAMO** [comment]
Done:
ex. 
``
http://localhost:3334/api/anda/table_data/?from_mjd=60662&to_mjd=60662.03&table_name=689_cav_temp_stab_I_TEC
``
response:
``
{
    "table_name": "689_cav_temp_stab_I_TEC",
    "from_mjd": "60662",
    "to_mjd": "60662.03",
    "data": {
        "mjd": [
            60662.0026502,
            60662.0057375,
            60662.0088269,
            60662.0119147,
            60662.0150016,
            60662.0180894,
            60662.0211789,
            60662.0242694,
            60662.0273594
        ],
        "val": [
            1236.52631,
            1236.37139,
            1236.25295,
            1236.29939,
            1236.52427,
            1236.37871,
            1236.62189,
            1236.67745,
            1236.75599
        ]
    }
}
``

**Comment by KLFAMO** [comment]
Let me know it it is ok or not.

**Comment by H0fek** [comment]
Implemented, works. My only suggestion is to change:
```http://localhost:3334/api/anda/table_data/?from_mjd=60662&to_mjd=60662.03&table_name=689_cav_temp_stab_I_TEC```
to
```http://localhost:3334/api/anda/table_data?from_mjd=60662&to_mjd=60662.03&table_name=689_cav_temp_stab_I_TEC```
In the first case we have an endpoint without a "name", it works under url /api/anda/table_data/.
In the second case we have an endpoint table_data which accepts supplied parameters and it works under /api/anda/.

**Comment by KLFAMO** [comment]
changed, resolve if ok
